integration-tests/deployment/devenv/rmn_config.go

```diff
@@ -10,8 +10,8 @@ import (
 
 const (
 	DefaultAFNPasphrase       = "my-not-so-secret-passphrase"
-	DefaultRageProxy          = "127.0.0.1:8081"
-	DefaultProxyListenAddress = "127.0.0.1:8080"
+	DefaultRageProxy          = "0.0.0.0:8081"
+	DefaultProxyListenAddress = "0.0.0.0:8080"
```

Comment:
Comments on what these ports are for would be useful to understand test setup. DefaultRageProxy is the afn <=> proxy server and DefaultProxyListenAddress is the P2P listen address? Other way around?

Reply:
Or maybe better naming, e.g `DefaultProxy2AFNAddress` and `DefaultProxyP2PListenAddress`